sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/PiiEntity.java

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+
+/**
+ * The {@link PiiEntity} model.
+ */
+@Immutable
+public final class PiiEntity {
+    /*
+     * PiiEntity text as appears in the request.
+     */
+    private final String text;
+
+    /*
+     * PiiEntity category, such as Person/Location/Org/SSN etc
+     */
+    private final EntityCategory category;
```

Comment:
Do you think we should have a specific enum for PII entity categories?

Reply:
Agreed. We could have a PiiEntityCatogry type since the category type varies. Also we could add HealthEntityCategory for the health endpoint as well in the future.
https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal

Reply:
hmmm I like it... my worry is if endpoints start mixing what type of entities they return.
For example, if PPI returns HealthCareEntity then this approach won't work...

When in PII we set domain... are those protected health information entities different than the ones in HealthCareEntity?

Reply:
deleted all domain related.